helpers/inbound/inbound_test.go

```diff
@@ -49,6 +49,14 @@ func TestParse(t *testing.T) {
 	}
 }
 
+func TestAttachments(t *testing.T) {
+	req := createRequest("./sample_data/raw_data_with_attachments.txt")
+	email := Parse(req)
+	contentType := "image/jpeg; name=\"TwilioSendGrid.jpg\""
+
+	assert.Equalf(t, contentType, email.Attachments[0].Headers["Content-Type"],"Expected From: %s, Got: %s", contentType, email.Attachments[0].Headers["Content-Type"])
```

Comment:
I'm not sure how this test is passing. I don't see where in the request an image/jpeg content type is set, since we are using a .txt file in this test.

Reply:
The [`createRequest` helper function](https://github.com/sendgrid/sendgrid-go/pull/416/files#diff-01fbecc73db32313c8fc9cbaf6644bc445e7ff4b2b9701d04f8dbfa42f30d548R13) builds a testing `Request` object from our included test emails. Within the test email example that includes an attachment (whose raw content [lives in `raw_data_with_attachments.txt`](https://github.com/sendgrid/sendgrid-go/blob/main/helpers/inbound/sample_data/raw_data_with_attachments.txt)) is an attachment with the [`"image/jpeg"` Content-Type](https://github.com/sendgrid/sendgrid-go/blob/main/helpers/inbound/sample_data/raw_data_with_attachments.txt#L33). This test verifies that we are able to parse that attachment's proper content type from the raw body of an Inbound Email request.